### PR TITLE
Framework: update gridicons to 2.1.2

### PR DESCRIPTION
**Pull request by enejb**
Update the gridicons to version [2.1.2](https://github.com/Automattic/gridicons/releases/tag/v2.1.2) that include the new plans icons. 
This should fix 
screen shot 2018-01-22 at 1 43 15 pm
> 
screen shot 2018-01-22 at 2 54 07 pm

The new updated gridicons were also added to http://calypso.localhost:3000/devdocs/design/gridicons.

screen shot 2018-01-22 at 3 09 33 pm

### CODE REVIEW

**Comment by enejb** [comment]
cc @MichaelArestad

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -6089,8 +6092,8 @@
       }
     },
     "gridicons": {
-      "version": "2.1.1",
-      "integrity": "sha1-dGETKidgO6i80L3mtDUZdo3MCNs=",
+      "version": "2.1.2-alpha.1",
```
Comment (gwwar) [inline npm-shrinkwrap.json]: wrong version here, we'll want to re-shrinkwrap

Comment (gwwar) [reply npm-shrinkwrap.json]: try `npm install gridicons@latest --save` and `npm run update-deps`

**Comment by gwwar** [comment]
Should be good to 🚢 after the shrinkwrap is fixed

**Pull request merged**